test/bdd/features/setupfabric.feature

```diff
@@ -65,11 +65,11 @@ Feature:
     And the JSON path "3.Name" of the response equals "peer0-org3.trustbloc.local.json"
     And the JSON path "3.ContentType" of the response equals "application/json"
 
-    Then we wait 10 seconds
+    Then we wait 60 seconds
 
     # Resolve .well-known files
-    When an HTTP request is sent to "https://peer1-org2.trustbloc.local/.well-known/did-trustbloc/consortium.json"
-    Then response from "https://peer1-org2.trustbloc.local/.well-known/did-trustbloc/consortium.json" to client contains value "payload"
+    When an HTTP request is sent to "https://peer1-org2.trustbloc.local/.well-known/did-trustbloc/testnet.trustbloc.local.json"
```

Comment:
can you change it to peer0-org1.trustbloc.local.json

Reply:
This won't resolve the problem (as we discussed)